src/client/components/UserListFilters/UserListFilters.tsx

```diff
@@ -24,13 +24,36 @@ const UserListFilters: React.FC = () => {
 
   const [modalOpen, setModalOpen] = useState(false)
 
+  const [userName, setUserName] = useState('')
+
+  useEffect(() => {
+    const delayDebounceFn = setTimeout(() => {
```

Comment:
Does this work like a real debounce or the action is dispatched after 1 second at each change ?
Would it make sense to dispatch the fetch instead with debounce ?(even when other filters change )